README.md

```diff
@@ -153,6 +153,7 @@ NOTE: You do not need to specify *end* segment - it will be added automatically.
 |--------|-------|-------|
 |`BULLETTRAIN_VIRTUALENV_BG`|`yellow`|Background color
 |`BULLETTRAIN_VIRTUALENV_FG`|`white`|Foreground color
+|`BULLETTRAIN_VIRTUALENV_NAME`|` %m`|Text to show after the prefix (use %m for virtualenv name)
```

Comment:
Isn't it enough to add space here
`local virtualenv_name=${BULLETTRAIN_VIRTUALENV_NAME:gs/%m/"${virtualenv_name}"}`
?